services/movie.py

```diff
@@ -1,11 +1,15 @@
+from django.db import transaction
 from django.db.models import QuerySet
 
+# import init_django_orm  # noqa 401
+
 from db.models import Movie
 
 
 def get_movies(
-    genres_ids: list[int] = None,
-    actors_ids: list[int] = None,
+        genres_ids: list[int] = None,
+        actors_ids: list[int] = None,
+        title: str = None
```

Comment:
add an annotation for None according to the previous comment. Also in all other similar cases